### PR TITLE
Default choises

### PR DESCRIPTION
**Pull request by Leefrost**
Small changes, just forgot to push

### CODE REVIEW

**Comment by rabingaire** [comment]
This all has been implemented please look at the master branch :) I have already merged your PR

**Comment by Leefrost** [comment]
Yeap, I read it before. In the last commit, I moved out default choices to dedicated massive and reuse it in the dialog. I just forgot to push it and back to it today. I have resolved the conflict. Sorry for late changes

File: index.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ const ora = require('ora');
 const TOKEN = require('./token');
 
 
+const lang = require('../lang/default')
```
Comment (rabingaire) [inline index.js:19]: It should be ./lang/default not ../lang/default

**Review by rabingaire — CHANGES_REQUESTED** [review]

**Comment by rabingaire** [comment]
And there has been a new update on code please pull it and push again sorry for trouble :)

**Comment by rabingaire** [comment]
There was no activity for a long time so I closed this PR

**Pull request closed without merging**